src/views/Correction.vue

```diff
@@ -1,36 +1,62 @@
 <template>
-  <div>
-    <section>
-      <header class="mt-4">
-        <h1>Correction - Incorporation Application</h1>
-      </header>
-
-      <div class="original-filing-date mt-6" v-if="isTypeBcomp">
-        <p>
-          <span class="original-filing-date-label">Original Filing Date:</span>
-          {{ filingDateLocal }}
-        </p>
-      </div>
-
-      <div class="benefit-company-statement mt-6" v-if="isTypeBcomp">
-        <p>
-          <span class="benefit-company-statement-label">{{ BenefitCompanyStatementResource.title }}:</span>
-          {{ BenefitCompanyStatementResource.description }}
-        </p>
-      </div>
-
-      <SummaryDefineCompany :isSummary="true" />
-      <!-- TODO: recognition date and time (as part of SummaryDefineCompany) -->
-      <!-- TODO: folio number (as part of SummaryDefineCompany) -->
-      <ListPeopleAndRoles :personList="getOrgPeople" :isSummary="true" />
-      <ListShareClass :shareClasses="getShareClasses" :isSummary="true" />
-      <AgreementType :isSummary="true" />
-      <!-- TODO: original completing party -->
-      <!-- TODO: 1. detail -->
-      <!-- TODO: 2. certify -->
-      <!-- TODO: 3. staff payment -->
-    </section>
-  </div>
+  <section>
```

Comment:
If you click the gear icon at the top of the page and hide whitespace changes, this is easier to review :)

Reply:
I didn't know that, thanks!